src/py42/sdk/queries/alerts/filters/alert_filter.py

```diff
@@ -29,7 +29,7 @@ def not_contains(cls, value):
 class DateObserved(QueryFilterTimestampField):
     """Class that filters alerts based on the timestamp the alert was triggered."""
 
-    _term = u"CreatedAt"
+    _term = u"createdAt"
```

Comment:
I recall you saying that there was a reason this was set to `CreatedAt`, is this correct and `CreatedAt` is just for the sort key?

Reply:
Correct, the "term" for the query is `createdAt` in both what the browser sends and what the response contains (although the endpoint does accept `CreatedAt`).

But the sort key has to be `CreatedAt` and does _not_ accept `createdAt`.